std/protocols/lookup.asm

```diff
@@ -62,9 +62,8 @@ let compute_next_z: Fp2<expr>, Fp2<expr>, Fp2<expr>, Constr, expr -> fe[] = quer
 /// the used challenges would overlap.
 /// TODO: Implement this for an array of constraints.
 /// Arguments:
-/// - lookup_constraint: The lookup constraint
-/// - multiplicities: A multiplicities column which shows how many times each row of the RHS value appears in the LHS                  
-let lookup: Constr, expr -> () = constr |lookup_constraint, multiplicities| {
+/// - lookup_constraint: The lookup constraint               
```

Comment:
Quite redundant, I would just remove it.

Reply:
done